tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/country_language_population.tsv

```diff
@@ -177,65 +177,67 @@ Cameroon	CM	"25,640,965"	71%	"89,540,000,000"		Ngomba	jgo	"87,500"	30%		http://w
+Canada	CA	"36,328,480"	99%	"1,774,000,000,000"	official_regional	Inuktitut	iu	0.114%	30%		https://www150.statcan.gc.ca/t1/tbl1/en/tv.action?pid=9810021601 Canada 2021 Census language "Knowledge of Language"; official status from Wikipedia Languages_of_Canada
+Canada	CA	"36,328,480"	99%	"1,774,000,000,000"	official_regional	Inuktitut (Latin)	iu_Latn	0.114%	30%		https://www150.statcan.gc.ca/t1/tbl1/en/tv.action?pid=9810021601 Canada 2021 Census language "Knowledge of Language"; official status from Wikipedia Languages_of_Canada
+Canada	CA	"36,328,480"	99%	"1,774,000,000,000"		Plautdietsch	pdt	0.113%			https://www150.statcan.gc.ca/t1/tbl1/en/tv.action?pid=9810021601 Canada 2021 Census language "Knowledge of Language"; official status from Wikipedia Languages_of_Canada
+Canada	CA	"36,328,480"	99%	"1,774,000,000,000"		Ojibwa	oj	0.0720%			https://www150.statcan.gc.ca/t1/tbl1/en/tv.action?pid=9810021601 Canada 2021 Census language "Knowledge of Language"; official status from Wikipedia Languages_of_Canada
+Canada	CA	"36,328,480"	99%	"1,774,000,000,000"		Oji-Cree	ojs	0.0421%			https://www150.statcan.gc.ca/t1/tbl1/en/tv.action?pid=9810021601 Canada 2021 Census language "Knowledge of Language"; official status from Wikipedia Languages_of_Canada
+Canada	CA	"36,328,480"	99%	"1,774,000,000,000"	recognized	Plains Cree	crk	0.0330%			https://www150.statcan.gc.ca/t1/tbl1/en/tv.action?pid=9810021601 Canada 2021 Census language "Knowledge of Language"; official status from Wikipedia Languages_of_Canada
+Canada	CA	"36,328,480"	99%	"1,774,000,000,000"	official_regional	Chipewyan	chp	0.0318%			https://www150.statcan.gc.ca/t1/tbl1/en/tv.action?pid=9810021601 Canada 2021 Census language "Knowledge of Language"; official status from Wikipedia Languages_of_Canada
+Canada	CA	"36,328,480"	99%	"1,774,000,000,000"	recognized	Innu-aimun	moe	0.0295%			https://www150.statcan.gc.ca/t1/tbl1/en/tv.action?pid=9810021601 Canada 2021 Census language "Knowledge of Language"; official status from Wikipedia Languages_of_Canada
+Canada	CA	"36,328,480"	99%	"1,774,000,000,000"		Mi'kmaq	mic	0.0254%			https://www150.statcan.gc.ca/t1/tbl1/en/tv.action?pid=9810021601 Canada 2021 Census language "Knowledge of Language"; official status from Wikipedia Languages_of_Canada
+Canada	CA	"36,328,480"	99%	"1,774,000,000,000"	recognized	Atikamekw	atj	0.0187%			https://www150.statcan.gc.ca/t1/tbl1/en/tv.action?pid=9810021601 Canada 2021 Census language "Knowledge of Language"; official status from Wikipedia Languages_of_Canada
+Canada	CA	"36,328,480"	99%	"1,774,000,000,000"		Siksika	bla	0.0183%			https://www150.statcan.gc.ca/t1/tbl1/en/tv.action?pid=9810021601 Canada 2021 Census language "Knowledge of Language"; official status from Wikipedia Languages_of_Canada
+Canada	CA	"36,328,480"	99%	"1,774,000,000,000"		Woods Cree	cwd	0.0140%			https://www150.statcan.gc.ca/t1/tbl1/en/tv.action?pid=9810021601 Canada 2021 Census language "Knowledge of Language"; official status from Wikipedia Languages_of_Canada
```

Comment:
The unit tests should prevent aliased language codes (like cwd). We need to fix that, but in the meantime you should use cr instead of cwd.

Reply:
Indeed, this change needs work.

I'd like to understand the macrolanguage better. As I've learned, Woods Cree `cwd` is being aliased to Cree `cr`. However, Woods Cree (5,110 speakers according to CA 2021 Census) isn't even the biggest Cree dialect -- Plains Creek `crk` is (12,005 speakers) -- however the vast majority of Cree speakers aren't grouped (there are 87,875 Cree speakers in total. 61,000 are not matched to a constituent dialect). So perhaps Woods Cree is actually the biggest.

So back to aliasing -- personally I'm not a big fan of that because they are grouping oranges with apples -- yes they are all fruits but they are different groups. Do you have any design docs about aliasing so I can understand the background?

For another example of aliasing losing data precision, when I worked on [CLDR-10478](https://unicode-org.atlassian.net/browse/CLDR-10478), the Macao census has different estimates for Traditional Chinese (written, any spoken dialect `zh_Hant`, at 98%), Simplified Chinese (written, any spoken dialect `zh_Hans`, at 5%) as well as Spoken Cantonese `yue` @ 86.2% and Spoken Mandarin `cmn` @ ~40%. I ended up just ignoring the Mandarin category because was being aliased back to `zh`.

Nonetheless, I have faith this has been thought-through a lot already, so maybe I just need to catch up to the design choices.